splitio/api/auth.py

```diff
@@ -42,7 +42,8 @@ def authenticate(self):
                 'auth',
                 '/v2/auth',
                 self._apikey,
-                extra_headers=self._metadata
+                extra_headers=self._metadata,
+                metric_name='token'
             )
             if 200 <= response.status_code < 300:
```

Comment:
you can track auth_rejection on the else statement instead of passing trough metric_name `token`

Reply:
and keep in mind that authrejections is only for 401 err

Reply:
fixed